sixtyfps_compiler/passes/repeater_component.rs

```diff
@@ -93,4 +93,18 @@ fn adjust_references(comp: &Rc<Component>) {
             };
         }
     });
+    // Transform any references to the repeated element to refer to the root of each instance.
+    visit_all_expressions(comp, |expr, _| {
```

Comment:
This is quite "aggressive" and I'm not entirely certain if this needs any additional checks ("access only from inside?").

Reply:
A few line earlier, we do something similar.  so i guess that's ok.